Domains/1-Common/AecUnits/AecUnits.ecschema.xml

```diff
@@ -3,7 +3,7 @@
 |  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 |  * See LICENSE.md in the project root for license terms and full copyright notice.
 ======================================================================================= -->
-<ECSchema schemaName="AecUnits" alias="AECU" version="01.00.04" description="This contains the unit definitions that are use across AE Schemas" displayLabel="Design Modeling Units" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+<ECSchema schemaName="AecUnits" alias="AECU" version="01.00.05" description="This contains the unit definitions that are use across AE Schemas" displayLabel="Design Modeling Units" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
```

Comment:
No need to increment the schema-version number. We keep WIP versions ahead of the latest released (in this case, the latest is 01.00.03).

Reply:
thanks, i've reverted the change

Reply:
That change-reverting also leads to reverting SchemaInventory.json.

Reply:
just the version right? just reverted

Reply:
i see, i've reverted the whole file